app/src/sharedTest/java/org/oppia/app/utility/TabMatcher.kt

```diff
@@ -1,12 +1,11 @@
 package org.oppia.app.utility
 
 import android.view.View
-import org.hamcrest.Description
-import org.hamcrest.TypeSafeMatcher
 import androidx.test.espresso.PerformException
 import com.google.android.material.tabs.TabLayout
+import org.hamcrest.Description
 import org.hamcrest.Matcher
-
+import org.hamcrest.TypeSafeMatcher
 
 /**
  * @see <a href="https://stackoverflow.com/questions/49626315/how-to-select-a-specific-tab-position-in-tab-layout-using-espresso-testing">TabMatcher</a>
```

Comment:
Also, updated this link to `https://stackoverflow.com/q/49626315` for better clarity. Normally we can make stack overflow links shorter but using the Id.